scripts/publish.sh

```diff
@@ -34,4 +34,10 @@ if [[ -n "$1" ]]; then
   fi
 fi
 
-yarn npm publish --tag "$PUBLISH_NPM_TAG"
+YARN_VERSION="$(yarn --version)"
+if [[ "$YARN_VERSION" =~ "^1" ]]; then
```

Comment:
Do we need the quotes around `^1`? I seem to recall that Bash treats the part after `=~` literally.

``` suggestion
if [[ "$YARN_VERSION" =~ ^1 ]]; then

Reply:
It's generally safest best-practice to always quote-wrap strings in bash even when not required.

Reply:
@mcmire FYI I have updated this PR after trying it out locally on a couple of repos and this is now more robust, inverted using a `-ge` on the major part.